lib/ibm_watson/discovery_v1.rb

```diff
@@ -1833,110 +1729,43 @@ def federated_query_notices(environment_id:, collection_ids:, filter: nil, query
     end
 
     ##
-    # @!method query_entities(environment_id:, collection_id:, feature: nil, entity: nil, context: nil, count: nil, evidence_count: nil)
-    # Knowledge Graph entity query.
-    # See the [Knowledge Graph
-    #   documentation](https://cloud.ibm.com/docs/services/discovery?topic=discovery-kg#kg)
-    #   for more details.
+    # @!method get_autocompletion(environment_id:, collection_id:, field: nil, prefix: nil, count: nil)
```

Comment:
prefix here should be required. did you regenerate with `ef9010ab7de1b1fc7628a340cab118e2f00dd51d`

Reply:
disregard this comment - i made it before i asked you about regenerating